.github/workflows/dialyzer.yml

```diff
@@ -0,0 +1,54 @@
+name: Dialyzer
+
+env:
+  ELIXIR_VERSION: "1.14.2"
+  MIX_ENV: dev
+  OTP_VERSION: "25.0.3"
+
+on:
+  push:
+    branches:
+      - "**"
+    paths-ignore:
+      - 'README.md'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  dialyzer:
+    name: Dialyzer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{ env.OTP_VERSION }}
+          elixir-version: ${{ env.ELIXIR_VERSION }}
+      - uses: actions/cache@v3
+        with:
+          path: deps
+          key: ${{ runner.os }}-mix-v2-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
+          restore-keys: |
+            ${{ runner.os }}-mix-v1-
+      - uses: actions/cache@v3
+        with:
+          path: _build/test
+          key: ${{ runner.os }}-test-build
+          restore-keys: |
+            ${{ runner.os }}-test-build
+      - name: Install Dependencies
+        run: mix deps.get
+      - name: Retrieve PLT Cache
+        uses: actions/cache@v3
+        id: plt-cache
+        with:
+          path: priv/plts
+          key: ${{ runner.os }}-dialyzer-plts-v4-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
+      - name: Create PLTs
+        if: steps.plt-cache.outputs.cache-hit != 'true'
+        run: |
+          mix dialyzer --plt
+      - name: Run dialyzer
+        run: mix dialyzer --no-check --halt-exit-status
```

Comment:
What's no-check?

Reply:
do not perform (quick) check to see if PLT needs update as we were updating it in previous step

https://github.com/jeremyjh/dialyxir/blob/master/lib/mix/tasks/dialyzer.ex#LL12C1-L12C1